SPEC.md

```diff
@@ -101,7 +101,9 @@ The key words "must", "must not", "required", "shall", "shall not", "should", "s
                     // * r - allows container to read from the specified device.
                     // * w - allows container to write to the specified device.
                     // * m - allows container to create device files that do not yet exist.
-                    "permissions": "<permissions>" (optional)
+                    "permissions": "<permissions>" (optional),
```

Comment:
Line 97 and 98 (`hostPath": "<path>",  "containerPath": "<path>",`) needs updating

Reply:
I haven't changed hostPath and containerPath. I've only added new fields.

Reply:
I think @RenaudWasTaken was suggesting that we drop these in favour of a single `path` while we are in a position to make breaking changes.

Reply:
This should be part of another PR I think.

Reply:
We can also agree to drop Permissions if favor of FileMode. This way our structure would be identical to the OCI LinuxDevice.

Reply:
+1 to that